project/profiles/capacity4more/modules/c4m/user/c4m_user_subscribe/c4m_user_subscribe.module

```diff
@@ -106,3 +106,47 @@ function _c4m_user_subscribe_entity_notifications_disabled($entity_id, $flag_suf
 
   return FALSE;
 }
+
+/**
+ * Implements hook_og_membership_type_insert().
+ *
+ * Flags the group subscription flag, so the user is subscribed to the group.
+ */
+function c4m_user_subscribe_og_membership_insert(OgMembership $membership) {
+  if ($membership->state == OG_STATE_ACTIVE) {
+    c4m_user_subscribe_to_group($membership);
+  }
+}
+
+/**
+ * Implements hook_og_membership_type_update().
+ *
+ * Flags the group subscription flag, so the user is subscribed to the group.
```

Comment:
@DavidHernandez 
if so, then change this line to `@see c4m_user_subscribe_og_membership_insert()`